docs/general/how-to-define-semantic-conventions.md

```diff
@@ -0,0 +1,151 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: How To Define New Semantic Conventions
+aliases: [how-to-define-new-semantic-conventions]
+--->
+
+# How to define new semantic conventions
+
+**Status**: [Development][DocumentStatus]
+
+<!-- toc -->
+
+- [Defining new conventions](#defining-new-conventions)
+  - [Best practices](#best-practices)
+    - [Defining attributes](#defining-attributes)
+    - [Defining spans](#defining-spans)
+    - [Defining metrics](#defining-metrics)
+    - [Defining resources](#defining-resources)
+    - [Defining events](#defining-events)
+- [Stabilizing existing conventions](#stabilizing-existing-conventions)
+  - [Migration plan](#migration-plan)
+
+<!-- tocstop -->
+
+This document describes requirements, recommendations, and best practices on how to define conventions
+for the new areas or make substantial changes to the existing ones.
+
+## Defining new conventions
+
+- New conventions MUST have a group of codeowners. See [project management](https://github.com/open-telemetry/community/blob/main/project-management.md) for more details.
+  <!-- TODO: add CI check for CODEOWNERS file (when a new area is added) -->
+- New conventions SHOULD be defined in YAML files. See [YAML Model for Semantic Conventions](/model/README.md) for the details.
```

Comment:
With this SHOULD I assume your intention is that things should be added to yaml files where possible, but also it's OK to add conventions as just text in markdown files? For example how to record span names? 

If this is right, maybe we can say something like this, to make it more clear?

> - New conventions MUST be defined in YAML files. Conventions that cannot be recorded as part of the YAML model MAY be added in the corresponding markdown file (e.g., conventions for span name)

Reply:
I want to put MUST here, but there are plenty of cases where we can't. Span names is a good example, but we have other things we can't yet express: references to metrics, complex attribute types, resources that are expected to be associated with telemetry, etc.

I use SHOULD for new conventions and MUST as a prereq to stabilization. 
I feel REQUIRING yaml for experimental stuff is not necessary, but also, it feels wrong to require something none of the conventions fully implement. 

Maybe once the tooling covers 90% of scenarios we can revisit it and be more strict?